packages/voter-dapp/package.json

```diff
@@ -9,7 +9,6 @@
     "@material-ui/styles": "^4.3.0",
     "@uma/common": "^2.0.1",
     "@uma/core": "^2.0.1",
-    "@uma/core-1-2-0": "npm:@uma/core@1.2.0",
```

Comment:
What did this binding do before?

Reply:
this was due to @mrice32 PR #2586. allowed us to load in new and old version of contract into app so we had both abis avaialble, or something like that